### ISSUE TITLE
Dotnet migrate has error in sln

### ISSUE THREAD

**Issue by onovotny** [dotnet/cli#5343]
- checkout rx master: https://github.com/Reactive-Extensions/Rx.NET
- go to \RxNet\Source
- delete global.json
- run dotnet migrate .\Rx.NET.sln

14 projects migrated
Error with sln

```
C:\dev\git\RxNet\Rx.NET\source [vs2017-3]> dotnet --version
1.0.0-rc3-004504
C:\dev\git\RxNet\Rx.NET\source [vs2017-3 +0 ~0 -1 !]> dotnet migrate .\Rx.NET.sln
Project System.Reactive.Core migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.Core)
Project System.Reactive.Interfaces migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.Interfaces)
Project System.Reactive.Linq migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.Linq)
Project System.Reactive.PlatformServices migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.PlatformServices)
Project System.Reactive.Observable.Aliases migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.Observable.Aliases)

Project System.Reactive.Providers migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.Providers)
Project System.Reactive.Experimental migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.Experimental)
Project System.Reactive.Windows.Threading migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.Windows.Threading)
Project System.Reactive.Runtime.Remoting migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.Runtime.Remoting)
Project System.Reactive.Windows.Forms migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.Windows.Forms)
Project System.Reactive.WindowsRuntime migration succeeded (C:\dev\git\RxNet\Rx.NET\source\System.Reactive.WindowsRuntime)
Project Microsoft.Reactive.Testing migration succeeded (C:\dev\git\RxNet\Rx.NET\source\Microsoft.Reactive.Testing)
Project Tests.System.Reactive migration succeeded (C:\dev\git\RxNet\Rx.NET\source\Tests.System.Reactive)
Project Tests.System.Reactive.Uwp.DeviceRunner migration succeeded (C:\dev\git\RxNet\Rx.NET\source\Tests.System.Reactive.Uwp.DeviceRunner)
Summary
Total Projects: 14
Succeeded Projects: 14
Failed Projects: 0

Solution C:\dev\git\RxNet\Rx.NET\source\Rx.NET.sln already contains project Tests.System.Reactive.Uwp.DeviceRunner\Tests.System.Reactive.Uwp.DeviceRunner.csproj.
```

**Comment by livarcocc** [comment]
I believe this may be a dup of https://github.com/dotnet/cli/issues/5310.

@jgoshi can you confirm?

**Comment by piotrpMSFT** [comment]
This was a dupe of #5345, but also exhibits another issue which I'm looking at now

**Comment by piotrpMSFT** [comment]
@onovotny looked at this migration after yesterday's changes and found that it now works succesfully. @onovotny can you close the issue if there are no lingering problems/

**Comment by livarcocc** [comment]
@onovotny I believe this is a flavor of https://github.com/dotnet/cli/issues/5310. @jgoshi, can you take a quick look and confirm?

**Comment by jgoshi** [comment]
This is a separate issue

**Comment by piotrpMSFT** [comment]
@onovotny @jgoshi if this is a separate issue, can you please file?

**Comment by jgoshi** [comment]
This issue was fixed by the referenced PR #5393
